src/Amadevus.RecordGenerator.Generators/RecordPartialGenerator.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
-using System;
```

Comment:
This change is out-of-context. Please revert.

Reply:
I'd like to refer to the boy scout rule here: [Clean Code, Robert C. Martin](http://www.informit.com/articles/article.aspx?p=1235624&seqNum=6&source=post_page). It doesn't belong in this Pull-Request but does a small clean-up, so why not take it?

Reply:
This was mostly because of #71/#74 which will probably go into conflict with these changes. Up to you. Resolve the comments however you see fit.